sync/fromDisk/assessments.js

```diff
@@ -167,6 +172,16 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
                     tries_per_variant: alternative.triesPerVariant,
                     question_id: questionId,
                     number_in_alternative_group: alternativeIndex + 1,
+                    min_advance_perc: alternative.minAdvancePerc,
+                    effective_min_advance_perc: [
+                            question.minAdvancePerc,
+                            alternativeGroupParams.min_advance_perc,
```

Comment:
Why is this `alternativeGroupParams` and not `alternative`?

Reply:
I think it's because we would consider it unfair if two students got two different alternative questions with different advanceScorePercs, but I just looked at the infoAssessment schema, and according to that, setting different advanceScorePercs for alternatives in the same group is valid. Your preference which one we should stick with.